### ISSUE TITLE
Multi project CI/CD deploy process to SaaS sandbox tried to deploy Test app

### ISSUE THREAD

**Issue by pzentner** [microsoft/AL-Go#282]
We have a few PTEs in a repo that is separated into two projects. During the deploy process (deploying it to a sandbox in SaaS), it deploys the apps fine and then tries to deploy the test apps. This fails, because we are using some dependencies that are not available on SaaS, but are needed to run our tests on the pipelines.

The apps and test apps are properly defined in the project specific .AL-Go/settings.json.

I don't think the workflow should deploy test apps or it should be able to be turned off.

This is currently happening in the preview version of AL-Go(-PTE).

**Comment by freddydk** [comment]
It definitely should not deploy test apps.
If you look inside the artifacts - are the test apps in the **project-branch-apps-versionnumber** artifact?
or are they in the project-branch-testapps-versionnumber artifacts?

**Comment by pzentner** [comment]
The artifacts look correct. It has the regular apps in the "-apps-version" artifact and the test apps in the "-TestApps-version" artifact. 
I just added you to the repo, in case you need to look at anything yourself. Not sure, if I did anything wrong - it's the first time I tried a multi project repo.

**Comment by freddydk** [comment]
Issued a hotfix for this.
Now it only publishes the apps (not the test apps)
Reason for this was confusion on the naming of the projects.
The publishing still fails (I re-ran the job), but this seems to be something different.

**Comment by pzentner** [comment]
Thank you. I will look at it.

One thing I noticed at some point was that, if you rerun the deploy job only and the same version of the app is installed already, it fails because the app is already installed. Not sure, if that's the same, just wanted to mention it.

Will report back, if there are any more issues that I think could be related to AL-Go and not my work

**Comment by freddydk** [comment]
I think that I did fix that issue in ContainerHelper.

**Comment by pzentner** [comment]
As I suspected, it's because of an existing app:

An extension with app Id '1af0fdd5-a507-4ce0-abde-55e042722824' and version at least '1.0.12.0' is already installed on this environment.

I removed the installed apps and ran it again and it succeeded. So, this issue is solved.

If the issue with the apps is "only" solved in the ContainerHelper preview so far, that might be why it's not working for me.
